tests/fixtures/netex_france/output/reseau_TheGreatNetwork_1e97c33560621530a594ced114597ea1/offre_f31e1eef20f64733a18c538073e78396.xml

```diff
@@ -106,14 +106,14 @@
 							<ForAlighting>true</ForAlighting>
 							<ForBoarding>true</ForBoarding>
 						</StopPointInJourneyPattern>
-						<StopPointInJourneyPattern id="FR:StopPointInJourneyPattern:M1F1_2:" order="3" version="any">
-							<ScheduledStopPointRef ref="FR:ScheduledStopPoint:M1F1_2:">
+						<StopPointInJourneyPattern id="FR:StopPointInJourneyPattern:M1F1_3:" order="4" version="any">
```

Comment:
Interesting (and kind of funny too), if we had chosen to make the filtering of route points at the reading of NTFS (instead of at the writing of NeTEx France), then `order` would not have been modified... This almost means that we're revealing an implementation detail here... But I believe it's fine!